docs/proposals/versioning.md

```diff
@@ -0,0 +1,25 @@
+## Plan for versioning APBs
+
+### Current status:
+* Currently all APBs are 0.1.0
+  * This is stored in a label on the image titled “com.redhat.apb.version”
+* Plan is to bump all APBs to a newer version to coincide with 3.7 Broker
+
+### Questions:
+* Should this be a major version bump? I.e. 1.0.0, or minor would be 0.2.0
+* Do we tag the images with the version of the broker or APB spec version? Should they match?
```

Comment:
Not sure if we do the versions by tags or if it's in the apb image name. The version will be in the label right? Can we change labels as easily as tags?

Reply:
The version will be in the label yes, but the broker doesn't have an easy way to see that version without introspecting the manifest. I think it makes sense to tag the image with whatever version is on the label.

Reply:
If I understand how the registry adapters are working already, then they are already using introspection to create the APB Spec struct. This is not me advocating against tagging + labeling. I just see labels as more rigid than tags and therefore more reliable as an indicator of what version the APB actually is; or said another way, you can tag an image as `v1.0.0`, `latest`, or `3.7` but what matters w.r.t. the broker knowing what version of the APB tool built you is the label.